packages/js/src/Modules/Verto/BrowserSession.ts

```diff
@@ -351,6 +351,57 @@ export default abstract class BrowserSession extends BaseSession {
 
   /**
    * Return supported resolution for the given webcam.
+   *
+   * @param deviceId this is the deviceId from your webcam.
+   *
+   * ## Examples
+   *
+   * Case deviceId is null
+   *
+   * 1º if deviceId is null and you do not have a webcam connect to your computer,
+   * it will throw an error 'Requested device not found'.
+   *
+   * 2º if deviceId is null and you have one or more webcam connect to your computer,
+   * it will return a list of resolutions from the default one set up in your system.
```

Comment:
```suggestion
   * 1) if deviceId is null and you do not have a webcam connect to your computer,
   * it will throw an error 'Requested device not found'.
   *
   * 2) if deviceId is null and you have one or more webcam connect to your computer,
   * it will return a list of resolutions from the default one set up in your system.
```